composer.json

```diff
@@ -11,11 +11,11 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=5.3.3",
+        "php": ">=5.5.3",
         "doctrine/common": "~2.2",
         "doctrine/orm": "~2.2",
-        "symfony/security": "~2.1",
-        "symfony/options-resolver": "~2.1",
+        "symfony/security": "~2.8|~3.2",
```

Comment:
If this is to support Symfony 3 as well, then what about 3.0 and 3.1?

I'd suggest a change to: `^2.8|^3.0`